src/utils/allocators.zig

```diff
@@ -204,6 +210,17 @@ pub fn RecycleFBA(config: struct {
     };
 }
 
+pub const RecycleFBAMetrics = struct {
```

Comment:
@0xNineteen Your ideas on metrics that should be collected here will be appreciated.

Reply:
i think thats a great starting point -- we could also add a historgram of the sizes that are free